js/build.js

```diff
@@ -84,14 +85,14 @@
                 data.name = data.dataSourceQuery.columns.category;
                 result.dataSourceEntries.forEach(function(row, i) {
                   data.entries.push({
-                    name: row[data.dataSourceQuery.columns.category] || 'Category ' + (i + 1),
+                    name: row[data.dataSourceQuery.columns.category] || T('widgets.chart.donut.category') + ' ' + TN(i + 1),
                     y: parseInt(row[data.dataSourceQuery.columns.value], 10) || 0
                   });
                 });
                 break;
               case 1:
-                // Summarize data
-                data.name = 'Count of ' + data.dataSourceQuery.columns.column;
+                // Summarise data
```

Comment:
Always use US spelling where possible.

```suggestion
                // Summarize data
```